src/client/client.rs

```diff
@@ -392,6 +392,11 @@ where
         }
     }
 
+    /// Access the `Connect` implementation directly, bypassing the connection pool.
```

Comment:
```suggestion
    /// Get a reference to the `Connect` value of this `Client`.
```

I don't think it needs to mention bypassing the pool. When I read that, it made me think "wait, do I have to be careful when bypassing something?"